src/modules/fixEmbedTwitterVideo/fixEmbedTwitterVideo.module.ts

```diff
@@ -0,0 +1,136 @@
+import { ActionRowBuilder, ButtonBuilder, ButtonStyle, MessageType } from 'discord.js';
+import { z } from 'zod';
+
+import { createModule } from '../../core/createModule';
+import { resolveCatch } from '../../helpers/resolveCatch.helper';
+
+interface FXTwitterResponse {
+  code: number;
+  message: string;
+  tweet: {
+    media?: {
+      videos?: {
+        type: 'gif' | 'video';
+        url: string;
+      }[];
+    };
+  };
+}
+
+const modulePrefixButtonId = 'removetwittervideo-';
+const deleteBotAnswerPrefixButtonId = modulePrefixButtonId + 'deleteBotAnswer-';
+const deleteBotAnswerButtonId = deleteBotAnswerPrefixButtonId + '$authorIdMessage';
+const ignoreConfirmationButtonId = modulePrefixButtonId + 'ignoreBotButtons';
+
+const tweetRegex = /https?:\/\/(mobile\.)?twitter\.com\/(\S+)\/status\/(\d+)/g;
```

Comment:
please reuse the same behavior as the old module, having an array of matching / replacements is great to support multiple domains.
It's interesting since we have twitter.com and x.com

Reply:
For me it's useless, because I don't want to replace the content of the message. The bot will only reply with the fxtwitter link. 
So it's only the regex that you have to modify.

Reply:
okay no my bad I have to replace also twitter by fxtwitter, i will change this